docs/news.rst

```diff
@@ -22,6 +22,7 @@ Version updates
   **Sep 15, 2023** The latest version is now 4.0.0_.
     - Support for Python 3.7 has been dropped
     - Added support for Python 3.9 and 3.10
+    - Added support for Python 3.11
```

Comment:
**Sep 21, 2023** The latest version is now 5.0.0_.
    - Added support for Python 3.11